src/calculateRentalCost.js

```diff
@@ -5,6 +5,18 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  const baseCost = days * 40;
+  const weekRental = 7;
+  const minDaysDiscount = 3;
+
+  // Aplica desconto baseado nos dias
```

Comment:
The comment is in Portuguese, which might not be consistent with the rest of the codebase if the primary language is English. Consider translating it to English for consistency.